ci/build.sh

```diff
@@ -12,9 +12,9 @@ version_id=$(. /etc/os-release && echo $VERSION_ID)
 if [ "$id" == fedora ] && [ "$version_id" == 26 ]; then
     echo -e '[fahc]\nbaseurl=https://ci.centos.org/artifacts/sig-atomic/fahc/rdgo/build/\ngpgcheck=0\n' > /etc/yum.repos.d/fahc.repo
     # Until we fix https://github.com/rpm-software-management/libdnf/pull/149
-    sed -i -e 's,metadata_expire=6h,exclude=ostree ostree-devel ostree-libs ostree-grub2\nmetadata_expire=6h,' /etc/yum.repos.d/fedora-updates.repo
+    sed -i -e 's,metadata_expire=6h,exclude=ostree ostree-devel ostree-libs ostree-grub2\nmetadata_expire=1m,' /etc/yum.repos.d/fedora-updates.repo
```

Comment:
Just noticed we're specifying `metadata_expire` twice in this line.

Reply:
Yeah that one was wrong, I meant the one above.  Fixed ⬇️